### PR TITLE
Fix some cast-function-type warnings introduced in GCC 8

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: mate-panel/panel-menu-button.c [hunk 1]
```diff
@@ -376,8 +376,11 @@ panel_menu_button_menu_deactivated (PanelMenuButton *button)
 }
 
 static void
-panel_menu_button_menu_detacher	(PanelMenuButton *button)
+panel_menu_button_menu_detacher (GtkWidget *attach_widget,
+                                 GtkMenu   *menu)
```
Comment (sc0w) [inline mate-panel/panel-menu-button.c:380]: unused parameter 'menu'

Comment (rbuj) [reply mate-panel/panel-menu-button.c:380]: @sc0w done

**Comment by rbuj** [comment]
Usually, I use the command below to check that I don't add any new warnings when I fixed some ones.
```shell
FILE=$1
for warning in $(grep warning $FILE | grep -Po '\[\-W.*\]' | sed 's/[][]//g' | sort -u)
do
 type=${warning#"-W"}
 num=$(echo "grep $type $FILE | wc -l" | sh)
 echo "$warning $num"
done
```
Output
```shell
$ diff <(./warnings-file.sh mate-panel.make.log) <(./warnings-file.sh mate-panel/make.log)
2c2
< -Wcast-function-type 24
---
> -Wcast-function-type 5
7d6
< -Wmaybe-uninitialized 2
```

**Comment by raveit65** [comment]
Just a rebase to  master....

**Comment by rbuj** [comment]
@raveit65 I think it's fixed. Test logout/login, run the command below:
```shell
$ grep mate-panel ~/.xsession-errors
```

**Comment by rbuj** [comment]
I can't reproduce your log.

na

**Comment by raveit65** [comment]
> I can't reproduce your log.

You did again a force push, my log was with the 2 previous versions.

> Test: force panel redraw by changing the color of the panel.

Why?
I see the warnings tray-applets are starting or closing.
Did yo you test this?

Well, i will test again new third version of PR.

**Comment by raveit65** [comment]
@sc0w 
Are you concerns solved and can you approve it?

**Comment by raveit65** [comment]
@sc0w 
Why not merging it?
Do you see any other active reviewer?

**Pull request merged**